### PR TITLE
Enhancement/multiple videos 112

### PR DESCRIPTION
**Pull request by dernicolas**
Added videos multiple videos to artist, artwork and movement

### CODE REVIEW

File: app/src/app/pages/movement/movement.component.html [hunk 1]
```diff
@@ -22,6 +27,7 @@ <h4 class="mt-2 more" href="#collapseArtist" (click)="toggleDetails()">
           </div>
 
           <app-video [entity]="movement"></app-video>
+          <app-video *ngFor="let artwork of sliderItems.slice(0,10)" [entity]="artwork"></app-video>
```
Comment (jschurse) [inline app/src/app/pages/movement/movement.component.html]: Should this be 10 Videos?

Comment (dernicolas) [reply app/src/app/pages/movement/movement.component.html]: It shows all related videos to the displayed slider items. Most often there are max 1-2 videos. Do you think there is another way to do this?

**Comment by jschurse** [comment]
In the other components you slice

sliderItems.slice(0,1)

In this component you slice

sliderItems.slice(0,10)


Is this a typo or should this be 10 videos?

Nicolas <notifications@github.com> schrieb am Mi., 18. Dez. 2019, 09:14:

**Comment by dernicolas** [comment]
Yes 10 is right

**Pull request merged**